editor/editor_spin_slider.cpp

```diff
@@ -215,7 +210,7 @@ void EditorSpinSlider::_value_input_gui_input(const Ref<InputEvent> &p_event) {
 			}
 		}
 
-		if (k->is_ctrl_pressed()) {
+		if (k->is_command_or_control_pressed()) {
```

Comment:
While checking all of the modifications. I realized that this case is totally a case of snaping. I had put it on `CMD_OR_CTRL` as tooltip already said you should use command even though you had to press ctrl in fact. I'm wondering if i should remove it and move it back to `CTRL` or if it is a sign cmd should be used for snapping.